pset-functions.js

```diff
@@ -115,6 +194,21 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+// maybe use For While loop here?
+function isPalidrome(word) {
+   let result = true
+   for (i=0; i<word.length/2; i++) {
+      if ((result === true) && (word[i] === word[word.length-1-i])) {
+         //do nothing and continue the loop
+      } else {
+         result = false
+      }
+   }
+   return result
+}
+
+console.log(isPalidrome("run"))
```

Comment:
Lance, absolutely brilliant solution here! You are checking both ends and meeting in the middle (without using built in methods)! If you did want to make use of some of javascript's built in methods, please find an alternate solution [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L163)